aiozk/session.py

```diff
@@ -304,3 +315,5 @@ def event_dispatch(self, event):
             self.state.transition_to(States.LOST)
         if self.conn:
             await self.conn.close(self.timeout)
+        self.closing = False
```

Comment:
It isn't obvious why here we should set `self.closing = False`

Reply:
Because we finished `close` process

Reply:
![image](https://user-images.githubusercontent.com/192182/54924743-161b1e80-4f1e-11e9-9c81-57a146e665ef.png)
what about this one?

Reply:
If we left this property in state `closing=True`, we can't start session if it was closed. I'm not sure that this is correct behavior. This property used for handling concurrency issues when you attempt to close session which already in closing state